src/main/java/nl/thehyve/ocdu/controllers/UploadSessionController.java

```diff
@@ -101,6 +101,34 @@ public String username() {
         }
     }
 
+    @RequestMapping(value = "/deleteSession", method = RequestMethod.POST)
+    public ResponseEntity<?> deleteSubmissionById(@RequestParam(value = "id") String sessionId, HttpSession session) {
```

Comment:
Did you try changing the parameter type from String to int? if that works you can avoid Integer.parseInt()